controller/state/statedb.go

```diff
@@ -123,39 +122,40 @@ func (s *stateDB) db() *sql.DB {
 }
 
 // Get returns a specific task identified by ID
-func (s *stateDB) Get(ctx context.Context, taskID string) (*tasks.Task, error) {
+func (s *stateDB) Get(ctx context.Context, taskID string) (tasks.Task, error) {
 	var serialized string
```

Comment:
hm, getting serialized bytes as a string is a bit weird. I'd add a TODO about using []byte directly here.